### PR TITLE
Add `bridge_parachains` pallet

### PR DESCRIPTION
**Pull request by jiguantong**

### CODE REVIEW

File: runtime/crab/src/pallets/bridge_grandpa.rs [hunk 1]
```diff
@@ -16,14 +16,18 @@
...
 
-pub use pallet_bridge_grandpa::Instance1 as WithDarwiniaGrandpa;
+pub use pallet_bridge_grandpa::Instance1 as WithPolkadotGrandpa;
 
 // darwinia
 use crate::*;
 
-impl pallet_bridge_grandpa::Config<WithDarwiniaGrandpa> for Runtime {
+frame_support::parameter_types! {
+	pub const PolkadotHeadersToKeep: u32 = 500;
```
Comment (AurevoirXavier) [inline runtime/crab/src/pallets/bridge_grandpa.rs:25]:
```
pub type PolkadotHeadersToKeep = ConstU32<500>;
```

**Review by AurevoirXavier — CHANGES_REQUESTED** [review]

File: runtime/crab/src/pallets/bridge_grandpa.rs (hunk 1 above)
Comment (jiguantong) [reply runtime/crab/src/pallets/bridge_grandpa.rs:25]: It needs to be used in two places, so I made it public and const.
https://github.com/darwinia-network/darwinia-2.0/blob/guantong%2Fbridge-pallets/runtime/crab/src/pallets/bridge_grandpa.rs#L30

https://github.com/darwinia-network/darwinia-2.0/blob/guantong%2Fbridge-pallets/runtime/crab/src/pallets/bridge_parachains.rs#L30

Comment (AurevoirXavier) [reply runtime/crab/src/pallets/bridge_grandpa.rs:25]: `pub type PolkadotHeadersToKeep = ConstU32<500>;`

is also a public item.

Comment (jiguantong) [reply runtime/crab/src/pallets/bridge_grandpa.rs]: ok i just misunderstood

**Review by fewensa — APPROVED** [review]

**Pull request merged**